packages/cli/src/cli.ts

```diff
@@ -28,6 +28,8 @@ export function runCli() {
     .command('dev')
     .description('Launch a local dev server')
     .argument('<file>', 'The file to serve')
+    .option('--port [port]', 'Specify dev server port number', '1234')
```

Comment:
Any reason why the port is optional? I don't think that should be the case, like with `--host`

Reply:
Thanks your feedback， My mistake, I'll change it.